### ISSUE TITLE
Provide official docker image

### ISSUE THREAD

**Issue by loveqianool** [fallaciousreasoning/progrssive#16]
Hi,

Very nice tool! It would be very helpful if one official docker image for this project would be provided. This would ease the integration with software architecture using containers and lower the barrier to experiment with it.

**Comment by fallaciousreasoning** [comment]
Hey there!  Thanks for commenting! I don't have much experience with Docker to be honest but my understanding of it was that it provides an image with the environment you need already set up?

This project only depends on [NodeJS](https://nodejs.org/en/download/) and [NPM](https://nodejs.org/en/download/), so I feel like Docker would probably be overkill here. However, if you want to run it inside a Docker image, it should work on any image with those dependencies.

So, to sum up:
I'm probably not going to add a Docker image sorry, I have no experience with Docker and don't feel like I need it (at least at the moment). That said, if you submit a PR adding support, I'd be happy to merge it :)

**Comment by loveqianool** [comment]
okay, thank you
